src/servers/src/grpc/greptime_handler.rs

```diff
@@ -136,10 +132,9 @@ pub(crate) async fn auth(
     }
     .map(Some)
     .map_err(|e| {
-        increment_counter!(
-            METRIC_AUTH_FAILURE,
-            &[(METRIC_CODE_LABEL, format!("{}", e.status_code()))]
-        );
+        METRIC_AUTH_FAILURE
+            .with_label_values(&[format!("{}", e.status_code()).as_str()])
```

Comment:
Can we use `to_string()` or derive [AsRefStr](https://docs.rs/strum_macros/0.25/strum_macros/derive.AsRefStr.html)?